pkg/resource/acl/hooks.go

```diff
@@ -114,17 +122,23 @@ func computeTagsDelta(
 	latest []*svcapitypes.Tag,
```

Comment:
It looks like we are reusing the `computeTagsDelta` in every `hooks.go` file. We can reduce this duplication by moving this method into a package that is accessible by all resources. RDS puts theirs in `pkg/utils/tags.go` and then imports that module inside `hooks.go` - https://github.com/aws-controllers-k8s/rds-controller/blob/main/pkg/util/tags.go

Reply:
Yeah, we can modify it after using latest code-generator to generate again.

Reply:
I will create another PR to do it.